source/manual/alerts/email-alerts-travel-medical.html.md

```diff
@@ -5,35 +5,35 @@ section: Icinga alerts
 subsection: Email alerts
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2020-07-16
+last_reviewed_on: 2020-08-18
 review_in: 6 months
 ---
 
-There is [a check][email-check] to verify that emails are sent for
-[drug and medical device alerts][] and [travel advice updates][]. These checks
-are run via Jenkins: [Drug and medical device alerts check][drug-alerts-check]
-and [Travel advice alerts check][travel-advice-check].
+We actively monitor [medical safety alerts][] and [travel advice updates][]
```

Comment:
This could use some info on what we actually monitor about those pages.

Reply:
Hmm I thought I could get away with whats below to describe what we're monitoring as I wasn't sure if the other stuff was needed.

```
These checks determine a list of content that we expect subscribers to have
been emailed. We use a Gmail account which is subscribed to all travel advice
and medical safety alerts to check whether an email has been received within a
sufficient time period, if not the checks will fail.
```

Have added in the extra detail though as per your suggestion! 👍